proc_nutation.py

```diff
@@ -1,53 +1,84 @@
+from pylab import *
 from pyspecdata import *
 from scipy.optimize import minimize
-from proc_scripts import hermitian_function_test,zeroth_order_ph,postproc_dict,fl_mod,center_CPMG_echo
+from proc_scripts import hermitian_function_test,zeroth_order_ph,postproc_dict,fl_mod
 from sympy import symbols
 from numpy import *
 fl = fl_mod()
 t2 = symbols('t2')
 logger = init_logging("info")
-for searchstr,exp_type,nodename,postproc in [
-    ['200219_nutation_alex_probe','test_equip','nutation','spincore_nutation_v1']
+max_kHz = 200
+for searchstr,exp_type,nodename,postproc,freq_slice,p90_varied in [
+    ['201211_Ni_sol_probe_nutation_1','nutation','nutation',
+        'spincore_nutation_v1',(-5000,13000),True],
+    #['210302_Ni_cap_probe_nutation_1','nutation','nutation',
+    #    'spincore_nutation_v1',(-4e3,1.2e4),True]
     ]:
     s = find_file(searchstr,exp_type=exp_type,expno=nodename,postproc=postproc,
-            lookup=postproc_dict) 
- 
+            lookup=postproc_dict)#,fl=fl) 
+    if not p90_varied:
+        plen = s.get_prop('acq_params')['p90_us']
+        plen *= 10**-6
     # {{{ do the rough centering before anything else!
     # in particular -- if you don't do this before convolution, the
     # convolution doesn't work properly!
-    s.ft(['ph2','ph1'])
-    # }}}
-    
+    s.ift('t2')
+    s.setaxis('t2', lambda x: x-abs(s['ph1',1]['ph2',-2]).mean_all_but('t2').argmax('t2').item())
```

Comment:
this is just a coarse correction -- why not just use Hermitian function test?

Reply:
my best guess is that you have more dimensions than it expects, and you end up running the zeroth order phasing for many many points